mypy/main.py

```diff
@@ -203,6 +204,16 @@ def add_invertible_flag(flag: str,
             strict_flag_names.append(flag)
             strict_flag_assignments.append((dest, not default))
 
+    def disallow_any_argument_type(raw_options: str) -> List[str]:
+        current_options = raw_options.split(',')
+        for option in current_options:
+            if option not in valid_disallow_any_options:
+                formatted_opts = ', '.join(map("'{}'".format, valid_disallow_any_options))
```

Comment:
If you're interested, talk to Guido about his views on `map` vs list comprehensions.  (Hint: they are very strong views.)